examples/sw/simple_system/pa_test/README.md

```diff
@@ -0,0 +1,89 @@
+#  Ibex Pointer Authentication Test
+
+Ibex Pointer Authentication Test is an application to test pac/aut instructions
+added for pointer authentication. This test confirms that pac/aut instructions can be executed correctly,
+and an exception occurs using pre-authentication pointers. 
+We are using Ibex Simple System, so see `examples/simple_system/README.md` for more details.
+
+## Prerequisites
+
+Basically, you need the prerequisites written `examples/simple_system/README.md`.
+However, please use the rv32imcb Tool Versions for RISC-V Compiler Toolchain.
+This test uses the 4-operand version of the instruction (this format is defined in bitmanip extension),
```

Comment:
"The aut instruction executed in this test uses the R4-type format (three source registers rs1, rs2, rs3 and one destination register rd). This format is only used by floating-point fused multiply-add (F) and some bit-manipulation (B) instructions".

Reply:
I added this description. Thank you!